TTHAnalysis/python/plotter/susy-sos-v2-clean/mca-includes/2018/3l_semiddbkg/mca-data.txt

```diff
@@ -1,9 +0,0 @@
-### 2018
-data : DoubleMuon_Run2018A_1Junel2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" , years="2018"
-data : DoubleMuon_Run2018B_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" , years="2018"
-data : DoubleMuon_Run2018C_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" , years="2018"
-data : DoubleMuon_Run2018D_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" , years="2018"
-data : MET_Run2018A_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" ,  years="2018"
-data : MET_Run2018B_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" ,  years="2018"
-data : MET_Run2018C_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" ,  years="2018"
-data : MET_Run2018D_1June2019	 ; FakeRate="susy-sos-v2-clean/fakerate/2018/2los/fakerate_appldata.txt, Label="Nonprompt e/\#mu" ,  years="2018"
```

Comment:
Should we just delete this file? I didn't notice doing the same for the other years, so I guess it was removed by mistake?